moja-kuhinjica-frontend/src/components/menu/MenuItem.tsx

```diff
@@ -2,17 +2,34 @@ import React, { useState } from 'react'
 import Image from 'next/image'
 import pic from '../../../public/static/assets/images/meal1.png'
 import styles from './MenuItem.module.scss'
+import { RegularButton } from '../button/RegularButton'
+import { Title } from '../label/Title'
+import { Text } from '../label/Text'
+import { AmountButton } from '../button/AmountButton'
 
-export const MenuItem = () => {
+interface IMenuItemProps {
+    type?: string
+}
+export const MenuItem = ({ type }: IMenuItemProps) => {
     const [openDescription, setOpenDescription] = useState(false)
 
+    const isOrdering = () => {
+        return type === 'ordering'
```

Comment:
You could just put here this without curly braces, then you also won't need return.
`const isOrdering = () => type === 'ordering' `

Reply:
Done.